File: proj/code-repair/mycode/clean_dataset/patch-single-line/quixbugs-python/gpt-neo-1.3/shunting_yard.py
--- 
+++ 
@@ -14,6 +14,7 @@
         else:
             while opstack and precedence[token] <= precedence[opstack[-1]]:
                 rpntokens.append(opstack.pop())
+            opstack.append(token)
 
     while opstack:
         rpntokens.append(opstack.pop())

File: proj/code-repair/mycode/clean_dataset/patch-single-line/quixbugs-java/gpt-j/shunting_yard.java
--- 
+++ 
@@ -17,6 +17,7 @@
             while (!opstack.isEmpty() && precedence.get(operator) <= precedence.get(opstack.getLast())) {
                 rpntokens.add(opstack.pop());
             }
+            opstack.push(token);
         }
 
     }
